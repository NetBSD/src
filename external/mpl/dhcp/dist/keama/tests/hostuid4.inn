# host declaration with client-identfiers config

# subnet4 declaration
subnet 10.5.5.0 netmask 255.255.255.224 {
    range 10.5.5.5 10.5.5.10;
}

# recommended when using dhcp-client-identifier options
ignore-client-uids false;

# host declarations
host test1 {
    uid 01:02:03:04:05:0a:0b:0c:0d:0e:0f;
    option domain-search "example.com", "example.org";
    default-lease-time 1800;
    fixed-address 10.5.5.1, 10.10.10.10;
}

host test2 {
    hardware ethernet 00:07:0E:36:48:19;
    fixed-address 10.5.5.2;
}

host test3 {
    hardware fddi 00:07:0E:36:48:19;
    fixed-address 10.10.10.1;
}

subnet 10.10.10.0 netmask 255.255.255.224 { }
