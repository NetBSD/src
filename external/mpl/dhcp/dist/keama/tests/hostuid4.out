{
  # host declaration with client-identfiers config
  # subnet4 declaration
  /// This configuration declares some subnets but has no interfaces-config
  /// Reference Kea #245
  "Dhcp4": {
    "subnet4": [
      {
        "id": 1,
        "subnet": "10.5.5.0/27",
        "pools": [
          {
            "pool": "10.5.5.5 - 10.5.5.10"
          }
        ],
        "reservations": [
          # host declarations
          {
            "hostname": "test1",
            "client-id": "01:02:03:04:05:0a:0b:0c:0d:0e:0f",
            "option-data": [
              {
                "space": "dhcp4",
                "name": "domain-search",
                "code": 119,
//              "original-data": "\"example.com\", \"example.org\"",
                "data": "example.com, example.org"
              }
            ],
//          /// default-valid-lifetime in unsupported scope
//          "valid-lifetime": 1800,
            "ip-address": "10.5.5.1"
//          "extra-ip-addresses": [
//            "10.10.10.10"
//          ]
          },
          {
            "hostname": "test2",
            "hw-address": "00:07:0e:36:48:19",
            "ip-address": "10.5.5.2"
          }
        ]
      },
      {
        "id": 2,
        "subnet": "10.10.10.0/27",
        "reservations": [
//        {
//          "hostname": "test3",
//          "hw-address": "fddi 00:07:0e:36:48:19",
//          "ip-address": "10.10.10.1"
//        }
        ]
      }
    ],
    "match-client-id": true,
    "host-reservation-identifiers": [
      "client-id",
      "hw-address"
    ]
  }
}
